File: SproInstall/SocketPro/tutorial/CSharp/SampleThree/Server/TThree_i.cs
public class TThreeConst
{
	//defines for service CTThree
	public const int sidCTThree = ((int)USOCKETLib.tagOtherDefine.odUserServiceIDMin + 10);

	public const short idGetOneItemCTThree = ((short)USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0);
	public const short idSendOneItemCTThree = (idGetOneItemCTThree + 1);
	public const short idGetManyItemsCTThree = (idSendOneItemCTThree + 1);
	public const short idSendManyItemsCTThree = (idGetManyItemsCTThree + 1);

    public const short idGetBatchItemsCTThree = (idSendManyItemsCTThree + 1);
    public const short idSendBatchItemsCTThree = (idGetBatchItemsCTThree + 1);
}

File: SproInstall/SocketPro/tutorial/VBNet/SampleThree/Server/TThree_i.vb
Public Class TThreeConst
	'defines for service CTThree
    Public Const sidCTThree As Integer = (CInt(USOCKETLib.tagOtherDefine.odUserServiceIDMin) + 10)

    Public Const idGetOneItemCTThree As Short = (CShort(USOCKETLib.tagOtherDefine.odUserRequestIDMin) + 0)
	Public Const idSendOneItemCTThree As Short = (idGetOneItemCTThree + 1)
	Public Const idGetManyItemsCTThree As Short = (idSendOneItemCTThree + 1)
	Public Const idSendManyItemsCTThree As Short = (idGetManyItemsCTThree + 1)

	Public Const idGetBatchItemsCTThree As Short = (idSendManyItemsCTThree + 1)
	Public Const idSendBatchItemsCTThree As Short = (idGetBatchItemsCTThree + 1)
End Class